instrumentation-genai/opentelemetry-instrumentation-openai-v2/tests/conftest.py

```diff
@@ -66,7 +66,12 @@ def openai_client():
 @pytest.fixture(scope="module")
 def vcr_config():
     return {
-        "filter_headers": ["authorization", "api-key"],
+        "filter_headers": [
+            ("cookie", "test_cookie"),
```

Comment:
here's the replacement and the way to add substitutions on request side (similar to how we do in response side)